config/country_dialing_codes.yml

```diff
@@ -1,5 +1,9 @@
 # NOTE: if updating this to match current AWS config, consider bringing back
 # the daily CircleCI check added in 3d4194c1f76ca1554852bacfde4abdea2ebb6104
+# `supports_voice` and `supports_sms` describe whether we are capable of sending
+# using those methods. A country that supports voice may be limited to
+# sending voice only to previously confirmed numbers when `supports_voice_unconfirmed` is set to false.
+# In those countries, it is not possible to register new phones.
```

Comment:
idea, maybe for a future PR

What if we added a separate config that was like `pinpoint_overrides` or something, so we have 1 YAML file that matches exactly what we generate from AWS, then another file where we explicitly override it to disable/enable things.

Then in this new overrides file, we can define things outside like `supports_voice_unconfirmed`?

Reply:
yeah, that's a great idea. Then we can enable the automatic check too.

Reply:
Filed https://cm-jira.usa.gov/browse/LG-4638 to track